src/main/kotlin/com/back/domain/auth/dto/request/MemberSignupRequest.kt

```
@@ -0,0 +1,16 @@
+
+data class MemberSignupRequest(
+    @field:NotBlank(message = "이메일은 필수입니다")
+    @field:Email(message = "올바른 이메일 형식이 아닙니다")
+    val email: String,
+
+    @field:NotBlank(message = "비밀번호는 필수입니다")
+    val password: String,
+
+    @field:NotBlank(message = "이름은 필수입니다")
+    val name: String
+){}
```

detekt (1.23.8)

[warning] The class or object MemberSignupRequest is empty.

(detekt.empty-blocks.EmptyClassBlock)